files/pt-br/web/accessibility/aria/roles/slider_role/index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Usando o slider role
+slug: Web/Accessibility/ARIA/Roles/slider_role
+translation_of: Web/Accessibility/ARIA/ARIA_Techniques/Using_the_slider_role
+original_slug: Web/Accessibility/ARIA/ARIA_Techniques/Using_the_slider_role
+---
+### Descrição
+
+Essa técnica demosntrará como usar o [slider](http://www.w3.org/TR/wai-aria/roles#slider) role.
+
+The` slider`` `role is used for markup that allows a user to select a value from within a given range. The slider role is assigned to the "thumb," the control that is adjusted to change the value. Typically, another element is styled to visually represent the range of possible valued, and the thumb positioned visually to represent the value within that range. As the user interacts with the thumb, the application must programmatically adjust the slider's `aria-valuenow` (and possible `aria-valuetext`) attribute to reflect the current value. See the [Examples](#examples) section below for more information.
```

Comment:
Hi @SphinxKnight,
Thank you for your contribution

Could you please fix the problemas below?

```suggestion
### Descrição

Essa técnica demonstrará como usar o [slider](http://www.w3.org/TR/wai-aria/roles#slider) role.

The `slider` role is used for markup that allows a user to select a value from within a given range. The slider role is assigned to the "thumb," the control that is adjusted to change the value. Typically, another element is styled to visually represent the range of possible valued, and the thumb positioned visually to represent the value within that range. As the user interacts with the thumb, the application must programmatically adjust the slider's `aria-valuenow` (and possible `aria-valuetext`) attribute to reflect the current value. See the [Examples](#examples) section below for more information.
```